MapboxNavigationTests/Support/FBSnapshotTestCase.swift

```diff
@@ -1,12 +1,15 @@
 import Foundation
 import FBSnapshotTestCase
 
+fileprivate let suffix: NSOrderedSet = ["_64"]
+
 @nonobjc extension FBSnapshotTestCase {
+    
     func verify(_ view: UIView, overallTolerance: CGFloat = 0.05) {
-        FBSnapshotVerifyView(view, suffixes: ["_64"], overallTolerance: overallTolerance)
+        FBSnapshotVerifyView(view, suffixes: suffix, perPixelTolerance: overallTolerance)
```

Comment:
perPixelTolerance != overallTolerance
Rename the parameter if you think perPixelTolerance is more appropriate.